### PR TITLE
Fix incorrect login info and add more notes

### PR DESCRIPTION
**Pull request by bergice**
Resolves https://github.com/silverstripeltd/product-issues/issues/289

For https://userhelp.silverstripe.org/en/4/managing_your_website/logging_in/

### CODE REVIEW

File: docs/en/01_Managing_your_website/01_Logging_In.md [hunk 1]
```diff
@@ -26,8 +26,18 @@ Enter your login (usually your email address) and password. This will take you t
 ![Login to Silverstripe CMS](../_images/general-login.png)
 
 [note]
-If you select the **Keep me signed in** checkbox field, the login fields will pre-populate the next time you login to the CMS. Only use this option on a computer where you alone have access.
+If you select the **Keep me signed in** checkbox field, you will be automatically signed in the next time you visit the site.
```
Comment (Cheddam) [inline docs/en/01_Managing_your_website/01_Logging_In.md:30]:
```suggestion
## Staying signed in ("Remember Me" functionality)

If you select the **Keep me signed in** checkbox field, you will be automatically signed in the next time you visit the site.
```

File: docs/en/01_Managing_your_website/01_Logging_In.md [hunk 2]
```diff
@@ -26,8 +26,18 @@ Enter your login (usually your email address) and password. This will take you t
...
+Only use this option on a computer where you alone have access.
 
+Additional information:
+
+* When checking this option you will stay signed in for 90 days by default.
+* You can stay signed in on multiple devices by default.
+* Logging out will clear automatic sign-ins on any device by default for your user which means you will need to log in again the next time.
+* Keep in mind the system administrator can change these default values.
```
Comment (Cheddam) [inline docs/en/01_Managing_your_website/01_Logging_In.md]:
```suggestion
* These features may behave differently based on how the developers / administrators have configured them.
```

File: docs/en/01_Managing_your_website/01_Logging_In.md [hunk 3]
```diff
@@ -26,8 +26,18 @@ Enter your login (usually your email address) and password. This will take you t
...
 [note]
-If you select the **Keep me signed in** checkbox field, the login fields will pre-populate the next time you login to the CMS. Only use this option on a computer where you alone have access.
+If you select the **Keep me signed in** checkbox field, you will be automatically signed in the next time you visit the site.
+Only use this option on a computer where you alone have access.
 
+Additional information:
+
+* When checking this option you will stay signed in for 90 days by default.
+* You can stay signed in on multiple devices by default.
+* Logging out will clear automatic sign-ins on any device by default for your user which means you will need to log in again the next time.
```
Comment (Cheddam) [inline docs/en/01_Managing_your_website/01_Logging_In.md]:
```suggestion
* Logging out of one device will clear automatic sign-ins on all of your devices by default, which means you will need to log in again next time.
```

**Review by Cheddam — CHANGES_REQUESTED** [review]
Good start, just a few language adjustments. Paul had also recommended pulling this information out of the note block and into its own subheading; I suggest doing the same with the lost password section.

**Comment by brynwhyman** [comment]
I see this has already been merged but this has overlap with the user help content that it's in the MFA section: https://github.com/silverstripe/silverstripe-mfa/blob/4/docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md

Given that it's not related to MFA functionality, it might make more sense to be removed from that section. Is there any content in the MFA docs that you think would be worth adding to this new section? Perhaps the screenshot and a little more detail on how the feature works? cc @silverstripeux @bergice

**Comment by clarkepaul** [comment]
Some of the content from "how trusted devices work" seems worthy in User Help. I couldn't think of any more content to add except maybe it would be good to elaborate a little... 
> there isn't any interface within the CMS for managing your remembered devices, but logging out of one device will clear automatic sign-ins on all of your devices by default. This means you will need to log in again next time.

**Pull request merged**